rllib/core/models/torch/primitives.py

```diff
@@ -67,96 +79,70 @@ def forward(self, x):
 
 
 class TorchCNN(nn.Module):
-    """A model containing a CNN and a final linear layer."""
+    """A model containing a CNN and a final linear layer.
+
+    Attributes:
+        input_dims: The input dimensions of the network.
+        cnn_filter_specifiers: A list of lists, where each element of an inner
+            list contains elements of the form
+            `[number of filters, [kernel width, kernel height], stride]` to
+            specify a convolutional layer stacked in order of the outer list.
+        cnn_activation: The activation function to use after each Conv2D layer.
+        cnn_use_layernorm: Whether to insert a LayerNorm functionality
+            in between each CNN layer's outputs and its activation.
+        output_activation: The activation function to use for the last filter layer.
+        use_bias: Whether to use bias on all Conv2D layers.
+    """
 
     def __init__(
         self,
-        input_dims: Union[List[int], Tuple[int]] = None,
-        filter_specifiers: List[List[Union[int, List]]] = None,
-        filter_layer_activation: str = "relu",
-        output_activation: str = "linear",
+        *,
+        input_dims: Union[List[int], Tuple[int]],
+        cnn_filter_specifiers: List[List[Union[int, List]]],
+        cnn_activation: str = "relu",
+        cnn_use_layernorm: bool = False,
+        use_bias: bool = True,
     ):
-        """Initialize a TorchCNN object.
```

Comment:
Can we unify TorchCNN and TfCNN docstrings?
What is now seen here as "Attributes" should be "Args", right?
I see similar stuff happening with TorchMLP!
Most of what is now listed in the class docstrings as attributes are not actually attributes but still args.

Reply:
completely unified these now.

Reply:
you are right, the primitives should have a `Args` list, not `Attributes`.
Fixed.